spec/cucumber/core/ast/examples_table_spec.rb

```diff
@@ -75,11 +75,16 @@ module Cucumber::Core::Ast
         end
       end
 
-      describe 'accesing the values' do
+      describe 'accessing the values' do
         it 'returns the actual row values' do
           row = ExamplesTable::Row.new({'x' => '1', 'y' => '2'}, 1, location, language, comments)
           expect( row.values ).to eq ['1', '2']
         end
+
+        it "can access a row value by it's parameter name" do
```

Comment:
So, I fixed one typo and introduced another. Truly, I am making this project a better place. :P